packages/parchment/lib/src/convert/html.dart

```diff
@@ -31,7 +31,7 @@ const _indentWidthInPx = 32;
 /// - default -> <p>
 /// - heading X -> <hX>
 /// - bq -> <blockquote>
-/// - code -> <pre><code>
+/// - code -> <pre>
```

Comment:
`<pre><code>` was decided according to this
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code#notes

To represent multiple lines of code, wrap the <code> element within a `<pre>` element. The <code> element by itself only represents a single phrase of code or line of code.

Reply:
Right - so I'm just using `<pre>` without nested `<code>` elements. I also kept `\n` characters in the `pre` which weren't there before. The result without the `code` element renders the same. For example, go to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre and within the "Try It" box, paste this:

```html
<pre >
  @override
  String convert(Delta input) {
    final state = _EncoderState();
    final inputOperations = input.toList();
    for (var i = 0; i &lt; inputOperations.length; i++) {
      final op = inputOperations[i];
    for (final op in input.toList()) {
      final buffer = state.buffer;
      final openInlineTags = state.openInlineTags;
</pre>
```
Notice how it is rendered. Now paste this:
```html
<pre >
<code>  @override</code>
<code>  String convert(Delta input) {</code>
<code>    final state = _EncoderState();</code>
<code>    final inputOperations = input.toList();</code>
<code>    for (var i = 0; i &lt; inputOperations.length; i++) {</code>
<code>      final op = inputOperations[i];</code>
<code>    for (final op in input.toList()) {</code>
<code>      final buffer = state.buffer;</code>
<code>      final openInlineTags = state.openInlineTags;</code>
</pre>
```
You'll see they're exactly the same. The former is just less verbose.

Reply:
I did some reading 

The `<pre>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents preformatted text which is to be presented exactly as written in the HTML file. The text is typically rendered using a non-proportional, or [monospaced](https://en.wikipedia.org/wiki/Monospaced_font), font. Whitespace inside this element is displayed as written.

The `<code>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element displays its contents styled in a fashion intended to indicate that the text is a short fragment of computer code. By default, the content text is displayed using the [user agent's](https://developer.mozilla.org/en-US/docs/Glossary/User_agent) default monospace font.

So it we want to display code we should use `<code>` and since we want to preserve format of text (new lines, spaces) we should embed it in `<pre>`

```html
<pre>
<code>
@override
  String convert(Delta input) {
    final state = _EncoderState();
    final inputOperations = input.toList();
    for (var i = 0; i &lt; inputOperations.length; i++) {
      final op = inputOperations[i];
    for (final op in input.toList()) {
      final buffer = state.buffer;
      final openInlineTags = state.openInlineTags;
<code>
</pre>

Does it make sense?

Reply:
Ok, it is a semantic difference, but I'll take care of it.

Reply:
Done